pages/api/openPR.js

```diff
@@ -152,62 +133,78 @@ function nomineeSubmission(values, sortedSubmission) {
     organizations: values.organizations ? [values.organizations] : [],
     stage: values.stage ? values.stage : "",
   };
-
   // Order nominee fields in the correct order e.g organizations
   sortedSubmission = orderFields(sortedSubmission);
   return sortedSubmission;
 }
 
+function submitPullRequests(projectName, filesObject) {
+  const MyOctokit = Octokit.plugin(createPullRequest);
+  const octokit = new MyOctokit({
+    auth: TOKEN,
+  });
+
+  // initialize an array for Promises
+  let promiseArray = [];
+
+  for (const submission in filesObject) {
+    let singleFile = {};
+    singleFile[submission.toString()] = filesObject[submission.toString()];
+
+    // Returns a normal Octokit PR response
+    // See https://octokit.github.io/rest.js/#octokit-routes-pulls-create
+    promiseArray.push(
+      composeCreatePullRequest(octokit, {
+        owner: GITHUB_OWNER,
+        repo: GITHUB_REPO,
+        title: submission.toString().includes("nominees")
+          ? `Add nominee: ${projectName}`
+          : `Add DPG: ${projectName}`,
+        body:
```

Comment:
@nathanfletcher add also a clause highlighting when a DPG is added.

Reply:
okay, what title should I use for that? And also, I thought this was the online form unless I'm missing something.

Reply:
When opening a pull request, the `body` tag generalizes to nominee whether its a nominee or dull submission, please see below:
`Automatic addition of a new nominee submitted through the online form available at https://digitalpublicgoods.net/submission
`
In order to cater for full submissions, updating the template text to highlight a DPG submission would be ideal. See text below for clarity:

`Automatic addition of a new DPG submitted through the online form available at https://digitalpublicgoods.net/submission`

Reply:
Resolved in commit bff7291. Looks good to me @nathanfletcher